Admin.js

```diff
@@ -114,561 +136,771 @@ var Admin = {
+                    elm.effect("highlight", {"color": "#57A957"}, 2000);
                     Admin.set_object_field_value(elm);
+
                 },
-                error: function(xhr, statusText, errorThrown) {
-                    jQuery(subject).parent().effect("highlight", {'color' : '#C43C35'}, 2000);
+                "error" (xhr, statusText, errorThrown) {
+
+                    jQuery(subject).parent().
+                        effect("highlight", {"color": "#C43C35"}, 2000);
+
                 }
             });
+
         });
+
     },
 
-    setup_collection_counter: function(subject) {
-        Admin.log('[core|setup_collection_counter] setup collection counter', subject);
+    "setup_collection_counter" (subject) {
+
+        Admin.log("[core|setup_collection_counter] setup collection counter", subject);
 
         // Count and save element of each collection
-        var highestCounterRegexp = new RegExp('_([0-9]+)[^0-9]*$');
-        jQuery(subject).find('[data-prototype]').each(function() {
-            var collection = jQuery(this);
-            var counter = 0;
-            collection.children().each(function() {
-                var matches = highestCounterRegexp.exec(jQuery('[id^="sonata-ba-field-container"]', this).attr('id'));
-                if (matches && matches[1] && matches[1] > counter) {
-                    counter = parseInt(matches[1], 10);
-                }
+        const highestCounterRegexp = new RegExp("_([0-9]+)[^0-9]*$");
+        jQuery(subject).find("[data-prototype]").
+            each(function () {
+
+                const collection = jQuery(this);
+                let counter = 0;
+                collection.children().each(function () {
+
+                    const matches = highestCounterRegexp.exec(jQuery("[id^=\"sonata-ba-field-container\"]", this).attr("id"));
+                    if (matches && matches[1] && matches[1] > counter) {
+
+                        counter = parseInt(matches[1], 10);
+
+                    }
+
+                });
+                Admin.lolCounters[collection.attr("id")] = counter;
+
             });
-            Admin.lolCounters[collection.attr('id')] = counter;
-        });
+
     },
 
-    setup_collection_buttons: function(subject) {
+    "setup_collection_buttons" (subject) {
+
+        jQuery(subject).on("click", ".sonata-collection-add", function (event) {
 
-        jQuery(subject).on('click', '.sonata-collection-add', function(event) {
             Admin.stopEvent(event);
 
-            var container = jQuery(this).closest('[data-prototype]');
-            var counter = ++Admin.lolCounters[container.attr('id')];
-            var proto = container.attr('data-prototype');
-            var protoName = container.attr('data-prototype-name') || '__name__';
+            const container = jQuery(this).closest("[data-prototype]");
+            const counter = ++Admin.lolCounters[container.attr("id")];
```

Comment:
- eslint: Combine this with the previous 'const' statement.
- eslint: Unary operator '++' used.